### DISCUSSION TITLE
Can't add submodule ananke

### DISCUSSION THREAD

**Discussion by ghost** [gohugo-ananke/ananke#720] [2 - Q&A / Support]
1. Added this to my config.toml file:
![Screen Shot 2022-12-27 at 10 04 26 am](https://user-images.githubusercontent.com/31978630/209588463-df207dad-9fb5-4a54-a52b-9bd15365c06a.png)

2. Typed this in the command line:
`hugo`

3. Got this error:
<img width="569" alt="Screen Shot 2022-12-27 at 10 04 05 am" src="https://user-images.githubusercontent.com/31978630/209588447-3392f023-ebd7-40d8-926c-d4bc0361112f.png">

Can someone please help me?

**Comment by anirban99** [comment]
When you add ananke as a git submodule, the following can be set in the config.toml

`theme = "ananke"`

**Comment by sebbASF** [comment]
Looks like the correct syntax requires square brackets:

```theme = ["github.com/theNewDynamic/gohugo-theme-ananke"]```

See

https://github.com/theNewDynamic/gohugo-theme-ananke#as-a-hugo-module-recommended

for all the details
